demo/dashboard/client/package.json

```diff
@@ -21,8 +11,20 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   },
+  "dependencies": {
+    "elasticsearch": "14.0.0",
+    "history": "4.7.2",
```

Comment:
I think you won't need this new dependency, please check the comment with some explanations.